upload/admin/model/extension/shipping/mds.php

```diff
@@ -0,0 +1,186 @@
+<?php
+
+use Mds\MdsAddress\MdsAddress;
+use Mds\MdsColliveryService;
+
+class ModelExtensionShippingMds extends Model
+{
+
+    private $columns = array(
+        'order' => array(
+            'collivery_from_address_id',
+            'collivery_from_contact_id',
+            'collivery_to_address_id',
+            'collivery_to_contact_id',
+            'collivery_town',
+            'collivery_suburb',
+            'collivery_location_type',
+            'collivery_price_data',
+            'collivery_service_type_id',
+            'waybill_id'
+        ),
+        'address' => array(
+            'collivery_town',
+            'collivery_suburb',
+            'collivery_location_type',
+        )
+    );
+
+    private $locationFields = array(
+        "10" => "Location Type",
```

Comment:
Please extract these "magic numbers" to constants

Reply:
9506f8a275d5255c010f78414ae54264066afb30

Reply:
290d06a2663a92863ecfd93936c0c6a002eeff87